spec/models/card_spec.rb

```diff
@@ -146,5 +146,21 @@
       end
     end
   end
+
+  context 'check levenshtein distance' do
+    before do
+      @card = user.cards.create(original_text: 'hello', translated_text: 'Привет',
+                                review_date: DateTime.current.to_date)
+    end
+
+    describe 'check error in translated text' do
+      before do
+        @errors_word = @card.check_translation('Прювет')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.